test/browsers.developer-extension.conf.js

```diff
@@ -0,0 +1,29 @@
+// Capabilities: https://www.browserstack.com/automate/capabilities
+
+const extensionBrowserConfigurations = [
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Can we use the `browsers.conf.js` and filter to only get Edge FF and Chrome. So we have only one place with browsers names and versions.